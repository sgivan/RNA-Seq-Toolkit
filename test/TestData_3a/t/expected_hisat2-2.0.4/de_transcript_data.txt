geneName	geneID	transcriptName	feature	id	fc	pval	qval	FPKM.sample01	FPKM.sample02	FPKM.sample03	FPKM.sample04	lg_fc
.	MSTRG.1	MSTRG.1.1	transcript	1	0.640632121680033	0.170749130996071	0.177328306454961	10103.432617	9821.423828	6936.536621	5856.232422	-0.642431957363798
.	MSTRG.2	MSTRG.2.1	transcript	2	1.15636797584292	0.170752361529111	0.177328306454961	3647.909424	9469.328125	6687.863281	5646.288086	0.209600560855707
.	MSTRG.3	MSTRG.3.1	transcript	3	0.744672261293962	0.170754247471528	0.177328306454961	42309.164062	52826.605469	37309.628906	31498.986328	-0.425322476076202
.	MSTRG.4	MSTRG.4.1	transcript	4	0.618262075544427	0.170749249437122	0.177328306454961	11083.049805	10155.375977	7172.395508	6055.358887	-0.693709582095153
.	MSTRG.5	MSTRG.5.1	transcript	5	0.744470588910407	0.170754125219056	0.177328306454961	36724.136719	45832.484375	32369.921875	27328.59375	-0.425713240131347
.	MSTRG.6	MSTRG.6.1	transcript	6	0.566205356958889	0.170747384358919	0.177328306454961	8058.763184	6379.015625	4505.28125	3803.623535	-0.820602696012012
.	MSTRG.7	MSTRG.7.1	transcript	7	0.699710843838517	0.170750033239193	0.177328306454961	10865.213867	12230.955078	8638.306641	7292.967285	-0.515169244751305
.	MSTRG.8	MSTRG.8.1	transcript	8	0.72867920688497	0.170752126879596	0.177328306454961	17265.714844	20792.902344	14685.317383	12398.209961	-0.456644271410676
.	MSTRG.9	MSTRG.9.1	transcript	9	0.621874050351961	0.170750208124064	0.177328306454961	14033.268555	12984.094727	9170.223633	7742.04248	-0.68530567744492
.	MSTRG.10	MSTRG.10.1	transcript	10	0.854029268163891	0.170750819894194	0.177328306454961	8087.664062	12681.972656	8956.844727	7561.895508	-0.227642582054064
.	MSTRG.11	MSTRG.11.1	transcript	11	1.1771098426781	0.170748556759146	0.177328306454961	4882.938965	13055.360352	9220.555664	7784.536133	0.235248952570841
.	MSTRG.12	MSTRG.12.1	transcript	12	0.737429463748958	0.170750816828044	0.177328306454961	11636.450195	14294.386719	10095.637695	8523.332031	-0.439423034903536
.	MSTRG.13	MSTRG.13.1	transcript	13	0.764225921908026	0.170755334261512	0.177328306454961	65680.851562	85620.382812	60470.761719	51052.976562	-0.387928901345423
.	MSTRG.14	MSTRG.14.1	transcript	14	0.658121127738369	0.170749457483282	0.177328306454961	10426.285156	10599.621094	7486.150879	6320.249023	-0.603574957271838
.	MSTRG.15	MSTRG.15.1	transcript	15	0.594553513999398	0.170747687954493	0.177328306454961	8214.314453	7052.584961	4981	4205.253418	-0.750121426368111
.	MSTRG.16	MSTRG.16.1	transcript	16	0.830567954542268	0.170751940799216	0.177328306454961	10091.723633	15108.193359	10670.401367	9008.581055	-0.267829884953229
.	MSTRG.17	MSTRG.17.1	transcript	17	0.630052562362203	0.170752156703146	0.177328306454961	35200.042969	33285.238281	23508.228516	19847.03125	-0.666455903900101
.	MSTRG.17	MSTRG.17.2	transcript	18	216.083239691499	0.170749676378458	0.177328306454961	0	15536.967773	10973.230469	9264.24707	7.75544336492594
.	MSTRG.18	MSTRG.18.2	transcript	20	1.1493436278734	0.0425801124801661	0.177328306454961	544732.5	426729.375	549038.125	589695.4375	0.200810195819011
.	MSTRG.19	MSTRG.19.1	transcript	21	0.735743383129125	0.170750339481348	0.177328306454961	10350.973633	12666.900391	8946.200195	7552.908691	-0.442725432468174
.	MSTRG.20	MSTRG.20.1	transcript	22	0.965325558448837	0.170760591401074	0.177328306454961	7536.620605	14488.288086	10232.583984	8638.949219	-0.0509125179469414
.	MSTRG.21	MSTRG.21.1	transcript	23	0.811742028636144	0.17075212340971	0.177328306454961	11241.204102	16199.520508	11441.168945	9659.308594	-0.300906782673284
.	MSTRG.22	MSTRG.22.1	transcript	24	0.853955748481307	0.170754341100877	0.177328306454961	12636.800781	19812.65625	13993.003906	11813.717773	-0.227766782770702
.	MSTRG.23	MSTRG.23.1	transcript	25	0.820209880936919	0.17075456702236	0.177328306454961	16864.408203	24726.316406	17463.355469	14743.591797	-0.285934971201717
.	MSTRG.24	MSTRG.24.1	transcript	26	0.825876158282251	0.170747068364869	0.177328306454961	5952.400391	8827.53418	6234.586426	5263.60498	-0.276002631919428
.	MSTRG.25	MSTRG.25.1	transcript	27	0.566470948073939	0.17074905281971	0.177328306454961	11671.467773	9246.401367	6530.418457	5513.36377	-0.819926126739915
.	MSTRG.18	MSTRG.18.1	transcript	19	1.04830208710831	0.627306971803427	0.627306971803427	39290.050781	78845.328125	56029.511719	52596.679688	0.0680545153223267
