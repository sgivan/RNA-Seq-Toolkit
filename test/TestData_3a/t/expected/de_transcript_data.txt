geneName	geneID	transcriptName	feature	id	fc	pval	qval	FPKM.sample01	FPKM.sample02	FPKM.sample03	FPKM.sample04	lg_fc
.	MSTRG.1	MSTRG.1.1	transcript	1	0.614327355330422	0.169198025016954	0.176268959257267	10188.942383	10102.762695	6856.97168	5684.093262	-0.702920468510722
.	MSTRG.2	MSTRG.2.1	transcript	2	25.37767468185	0.169194067656498	0.176268959257267	0	6452.781738	4379.647461	3630.513184	4.66548797814625
.	MSTRG.3	MSTRG.3.1	transcript	3	0.68604483692491	0.169204536766311	0.176268959257267	48035.285156	61813.761719	41954.386719	34778.132812	-0.543625227096606
.	MSTRG.4	MSTRG.4.1	transcript	4	0.574572874003434	0.169198031776739	0.176268959257267	13222.679688	11196.134766	7599.068359	6299.254883	-0.799438211074524
.	MSTRG.5	MSTRG.5.1	transcript	5	0.680566498635811	0.169203721353314	0.176268959257267	29661.388672	37453.628906	25420.619141	21072.447266	-0.555191959513846
.	MSTRG.6	MSTRG.6.1	transcript	6	0.5285389856843	0.169190647419358	0.176268959257267	5464.048828	3797.844727	2577.682373	2136.772461	-0.919918204397618
.	MSTRG.7	MSTRG.7.1	transcript	7	0.693385823462764	0.16920185518147	0.176268959257267	12283.583008	16208.210938	11000.876953	9119.1875	-0.528269754494647
.	MSTRG.8	MSTRG.8.1	transcript	8	0.755790710870524	0.169205928579752	0.176268959257267	14851.676758	24017.060547	16300.917969	13512.662109	-0.403941307783779
.	MSTRG.9	MSTRG.9.1	transcript	9	0.57898542493729	0.169181068536338	0.176268959257267	2519.181152	2170.638184	1473.260864	1221.261108	-0.788401063832361
.	MSTRG.11	MSTRG.11.1	transcript	11	1.09263122313497	0.169175551569973	0.176268959257267	2411.794922	9308.948242	6318.191895	5237.471191	0.127806555329919
.	MSTRG.12	MSTRG.12.1	transcript	12	0.705021401116598	0.169202848869799	0.176268959257267	13718.399414	18826.691406	12778.098633	10592.416992	-0.504261043310446
.	MSTRG.13	MSTRG.13.1	transcript	13	0.710977225733277	0.16920635624023	0.176268959257267	100464.078125	140645.21875	95459.0625	79130.882812	-0.492124747205665
.	MSTRG.14	MSTRG.14.1	transcript	14	0.487581730917247	0.169186526454357	0.176268959257267	4410.018066	2533.248779	1719.372803	1425.275757	-1.03628402391049
.	MSTRG.15	MSTRG.15.1	transcript	15	0.653988533754103	0.169181638590671	0.176268959257267	2248.349854	2583.037354	1753.165405	1453.288208	-0.612662753416465
.	MSTRG.16	MSTRG.16.1	transcript	16	20.741545471678	0.169194830367835	0.176268959257267	0	4007.765625	2720.1604	2254.879639	4.37445148958128
.	MSTRG.17	MSTRG.17.1	transcript	17	0.635347192013437	0.169201787177528	0.176268959257267	26974.085938	28958.779297	19654.972656	16293.009766	-0.654382911959665
.	MSTRG.17	MSTRG.17.2	transcript	18	26.2337044468787	0.169193970405864	0.176268959257267	0	6978.447754	4736.429199	3926.267578	4.71334963904323
.	MSTRG.18	MSTRG.18.2	transcript	20	1.14622583654026	0.0759417142992211	0.176268959257267	594427	488546.5	591445.5625	625225.75	0.196891320846741
.	MSTRG.19	MSTRG.19.1	transcript	21	0.619098939794377	0.169197681646797	0.176268959257267	9412.416992	9504.699219	6451.052246	5347.606445	-0.691758106216373
.	MSTRG.20	MSTRG.20.1	transcript	22	0.702579257201687	0.169202664464128	0.176268959257267	13598.458984	18509.873047	12563.066406	10414.166016	-0.509267111515256
.	MSTRG.21	MSTRG.21.1	transcript	23	0.758714359400206	0.169204324699873	0.176268959257267	9945.878906	16230.697266	11016.138672	9131.838867	-0.398371252513113
.	MSTRG.22	MSTRG.22.1	transcript	24	0.872970799923559	0.169201763746818	0.176268959257267	3313.659668	7529.636719	5110.533691	4236.381836	-0.19599469705787
.	MSTRG.23	MSTRG.23.1	transcript	25	0.762183471300156	0.169202468752217	0.176268959257267	7279.907715	12008.47168	8150.419434	6756.29834	-0.391789772662047
.	MSTRG.24	MSTRG.24.1	transcript	26	1.05043613364002	0.169218200886977	0.176268959257267	1281.661743	4508.241211	3059.844727	2536.461182	0.0709884490344723
.	MSTRG.18	MSTRG.18.1	transcript	19	1.20752234606563	0.301125168613617	0.301125168613617	27216.458984	54280.171875	48321.355469	49496.929688	0.272049887412823
